# Rocket parameters
stages: 4
diameters:
  - 0
  - 0.2 
  - 0.2 
  - 0.2
stage_z: 
  - 0 
  - 0.1 
  - 0.2 
  - 0.5 
fin_n: 3
fin_top_offset: 0.25 
fin_span: 0.1
fin_root_chord: 0.1 # base length
fin_tip_chord: 0.1 
fin_sweep: 0.1
fin_thickness: 0.004
lug_n: 2
lug_S: 5.7e-4 
dry_mass: 1.7
dry_CM: 0.1
dry_I:
  - 0.125
  - 0.125
  - 0.02
propellant_mass: 0.3 #3.693
propellant_CM: 0.1
Isp: 1000000
maxThrust: 30
minThrust: 0
Cd:
  - 2
  - 2
  - 0.7
